examples/Admin/policies.py

```diff
@@ -78,19 +78,18 @@ def bulk_delete_section(policy_keys, print_response=False):
         pretty = json.dumps(response, indent=4, sort_keys=True, default=str)
         print(pretty)
 
-def update_policy_with_device_health_app(policy_key, print_response=False):
+def update_policy_with_duo_desktop(policy_key, print_response=False):
```

Comment:
So I have many questions about backwards compatibility :)

How important is it to change the name of this method.  Removing the old method name would require a major version bump.  Possibly we could
a) introduce the new method
b) keep the old method
c) mark the old method as deprecated
d) have the old method simply call the new method
?

Reply:
Ahhh gotcha, its not critical, but its misleading to customers because device health app is the old name for Duo Desktop.

I think we could do the old method calling the new method and that would be fine!